docs/source/index.rst

```diff
@@ -80,6 +80,32 @@ python files. The recommended way to use this is with a context manager
     with adl.open('newfile', 'rb') as f:
         df = pd.read_csv(f) # read into pandas.
 
+To seamlessly handle remote path representations across all supported platforms,
+the main API will take in numerous path types: string, Path/PurePath, and
+AzureDLPath. On Windows in particular, you can pass in paths separated by either
+forward slashes or backslashes.
```

Comment:
The paths are just for remote files. For local paths, we keep the native path style, so `/my/file/for/upload.txt` would be a file in the current directory with forward slashes in the name.